docs/how-to/download-new-upstream-version.rst

```diff
@@ -1,9 +1,139 @@
+
+    :manpage:`uscan(1)` verifies downloads against this :term:`Signing Key`
+    automatically (if available).
+
+Download new :term:`Upstream` version (if available)
+----------------------------------------------------
+
+Running :manpage:`uscan(1)` from the :term:`Root` of the :term:`Source Tree` will
+check if a newer :term:`Upstream` version exits and downloads it:
+
+.. code:: bash
+
+    uscan
+
+If :manpage:`uscan(1)` could not find a newer :term:`Upstream` version it will
+return with the exit code `1` and print nothing to the :term:`Standard Output`.
+
+:manpage:`uscan(1)` reads the first entry in :file:`debian/changelog` to determine
+the name and version of the :term:`Source Package`.
+
+You can allways add the ``--verbose`` flag to see more information (e.g., which version
+:manpage:`uscan(1)` found):
+
+.. code:: bash
+
+    uscan --verbose
+
+Check for new :term:`Upstream` version (no download)
+----------------------------------------------------
+
+If you just want to check if a new update is available, but you don't want to download
+anything, you can run the :manpage:`uscan(1)` :term:`Command` with the ``--safe`` flag
+from the :term:`Root` of the :term:`Source Tree`:
+
+.. code:: bash
+
+    uscan --safe
+
+Force the download of the latest :term:`Upstream` version
+-----------------------------------------------------
+
+You can use the ``--force-download`` flag to download the latest :term:`Upstream` release
+from the :term:`Upstream` project, even if the :term:`Upstream` Release is up-to-date
+with the :term:`Source Package`:
+
+.. code:: bash
+
+    uscan --force-download
+
+.. note::
+
+    In most cases you actually want to download the :term:`Source` from the :term:`Archive`
+    and not re-download the :term:`Source` from the :term:`Upstream` project.
+    
+    :ref:`How to get the Source from the Archive? <get-package-source>`
+
+Download the :term:`Source` of older Versions from the :term:`Upstream` project
+-------------------------------------------------------------------------------
+
+If you want to download the :term:`Source` of a specific version from the
+:term:`Upstream` project you can use the ``--download-version`` flag.
+
+Basic syntax:
+
+.. code:: bash
+
+    uscan --download-version VERSION
+
+For example:
+
+.. code:: bash
+
+    uscan --download-version '1.0'
+
+In the special case that you want to download the :term:`Source` for the
+current version of the :term:`Source Package` from the :term:`Upstream` project
+you can use the ``--download-current-version`` flag instead, which parses the
+version to download from the first entry in :file:`debian/changelog` file:
+
+.. code:: bash
+
+    uscan --download-current-version
+
+.. note::
+
+    The ``--download-version`` and ``--download-current-version`` flags are
+    both a :term:`best-effort` features of :manpage:`uscan(1)`.
+    
+    There are special cases where they do not work for technical reasons.
+
+.. note::
```

Comment:
This note appears twice - is that intentional?

Reply:
Yes, this was intentional for readers that read the article only partially. I also don't think that it is elegant.

Do you have a more elegant solution?